pr-data.csv

```diff
@@ -6157,10 +6151,7 @@ https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-api,net.ripe.db.whois.api.rdap.RdapResponseJsonTest.nameserver_serialization_test,ID,Accepted,https://github.com/RIPE-NCC/whois/pull/1097,
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-endtoend,net.ripe.db.whois.spec.domain.AckResponseTest.successes,UD,,,
 https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1274,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1276,
 https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1272,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1274,
```

Comment:
Add https://github.com/robovm/robovm/pull/1274 into `Notes` for https://github.com/robovm/robovm/pull/1272